CONTRIBUTING.md

```diff
@@ -72,15 +72,15 @@ git commit -s
 Until we are able to set up a dedicated channel, please connect with us through https://chat.fhir.org/ or by opening an issue in our tracker.
 
 ## Setup
-The Watson Health FHIR Server is built with Maven and requires Java 8 or higher.  To build the project from the root directory, please execute:
+The `IBM Server for HL7 FHIR` is built with Maven and requires Java 8 or higher.  To build the project from the root directory, please execute:
```

Comment:
I don't think the name needs backticks, but I suppose its fine.

Reply:
no problem - either way is fine.  I highlighted to easily find in the future.